src/social/SocialIcon.js

```diff
@@ -82,22 +96,21 @@ const SocialIcon = props => {
   return (
     <Component
       {...attributes}
-      underlayColor={light ? 'white' : underlayColor || colors[type]}
-      onLongPress={disabled ? null : onLongPress || log}
-      onPress={(!disabled || log) && (onPress || log)}
+      {...touchableProps}
+      {...highlightProps}
       disabled={disabled || false}
       style={[
         raised && styles.raised,
         styles.container,
         button && styles.button,
         !button && raised && styles.icon,
         !button &&
-        !light &&
-        !raised && {
-          width: iconSize * 2 + 4,
-          height: iconSize * 2 + 4,
-          borderRadius: iconSize * 2,
-        },
+          !light &&
```

Comment:
Unnecessary indent added on lines 108-113

Reply:
I believe the indent is necessary.  Line 108 is a continuation of 107, just as 110 is a continuation of 109, and is indented